gammapy/irf/effective_area_table.py

```diff
@@ -444,6 +446,67 @@ def read(filename):
         hdu_list = fits.open(filename)
         return EffectiveAreaTable2D.from_fits(hdu_list)
 
+    def to_effectiveareatable(self, offset, energ_lo = None, energ_hi = None):
```

Comment:
Please run PEP8 on this file. There's some whitespace things that should be changed, e.g. it should be `energ_lo=None` without spaces instead of `energy_lo = None` with spaces for keyword arguments.